modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormPagesTemplateContextFactory.java

```diff
@@ -81,14 +81,15 @@ public DDMFormPagesTemplateContextFactory(
 		DDMFormValues ddmFormValues =
 			ddmFormRenderingContext.getDDMFormValues();
 
-		if ((ddmFormValues == null) ||
-			ListUtil.isEmpty(ddmFormValues.getDDMFormFieldValues())) {
```

Comment:
@rodrigopaulino what happens if the condition `ListUtil.isEmpty(ddmFormValues.getDDMFormFieldValues())` is true?

Reply:
For the cases where some class created a `DDMFormValues` object and directly called `setDDMFormFieldValues(null)`, it could cause an NPE. I'll put it back.